### ISSUE TITLE
Missing indentations

### ISSUE THREAD

**Issue by Einstone-edge** [vonfeng/DeepMove#5]
Hi, Jie 
Sorry to bother you. Your work is excellent, but I have encountered some minor problems when reading the code.
At train.py in codes document, when defining generate_input_long_history2, there may be missing an indentation at every line from 145 to 154.

**Comment by lmyacj2006** [comment]
+1

**Comment by gunarto90** [comment]
I agree but I think it is not only from 145 to 154 but it is from **145 to 159** because it is quite strange that the variable "i" is referred but it is not inside a loop. If it is designed like that, can the author verify the logic of it? Thank you
